modules/afsocket/afinet-dest.c

```diff
@@ -152,6 +189,23 @@ _determine_port(const AFInetDestDriver *self)
   return port;
 }
 
+static void
+_setup_next_hostname(AFInetDestDriver *self)
+{
+  gchar *next_dst_candidate = _get_next_destination_candidate(self);
+  if (strcmp(next_dst_candidate, self->hostname) == 0)
+    {
+      msg_warning("The server is inaccessible, trying the old host again", evt_tag_str("host", self->hostname));
+    }
+  else
+    {
+      msg_warning("Current server is inaccessible, sending the messages to the next failover server",
+                  evt_tag_str("current", self->hostname),
+                  evt_tag_str("failover", next_dst_candidate));
```

Comment:
Would it make sense to print the log_expr_node_location_tag as well? Maybe multiple writers can share the same failover list. (well in that case probably both will fail on this specific candidate, so again this is just a suggestion)

Reply:
both the id and the location would be useful, I think, not just here but in the rest of the messages.